src/components/ChatEntry.js

```diff
@@ -1,22 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateLikes }) => {
```

Comment:
👍 Nice use of deconstruction in the definition to make the props a little more explicit.